README.md

```diff
@@ -63,15 +63,59 @@ Then add `__tests__` to `sources` in your `bsconfig.json`:
 ```js
 "sources": [
   {
-    "dir": "src"
+    "dir": "app",
+    "subdirs": true
   },
   {
     "dir": "__tests__",
-    "type": "dev"
+    "type": "dev",
+    "subdirs": true
   }
 ]
 ```
 
+Then your project structure would look like this:
+
+```
+src/
+├── MyComponent.re
+__tests__/
+├── MyComponent_test.re
+README.md
+
+
+(NOTE: no .re files in the root of src, if you put them there they will not be built)
+```
+
+### For Create React App
```

Comment:
I still don't see the content of this section as good advice. Why not recommend `src/app` instead, and avoid all the warnings and such?